integration_test/tests/wallet.rs

```diff
@@ -310,6 +310,29 @@ fn wallet__import_address() {
     let _: () = node.client.import_address(&addr).expect("importaddress");
 }
 
+#[test]
+fn wallet__import_pruned_funds() {
+    let node = match () {
+        #[cfg(feature = "v22_and_below")]
+        () => Node::with_wallet(Wallet::Default, &["-txindex"]),
+        #[cfg(not(feature = "v22_and_below"))]
+        () => {
+            let node = Node::with_wallet(Wallet::None, &["-deprecatedrpc=create_bdb", "-txindex"]);
```

Comment:
```suggestion
    let node = Node::with_wallet(Wallet::Default, &["-txindex"]);
```
Why is this feature gated for version? importprunedfunds has the same functionality for all versions and should work on both legacy and descriptor wallets.